src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -29,11 +33,15 @@
         display: table-cell;
     }
 
-    @include basic-table(".datagrid", ".datagrid-head", ".datagrid-body",
-        ".datagrid-row", ".datagrid-column", ".datagrid-cell");
-
     $clr-datagrid-icon-size: rem(1);
 
+    .datagrid-content-wrapper {
+        display: block;
+        overflow-x: auto;
```

Comment:
Hey @jeeyun. I dont think we need to separate this anymore. We can directly use `overflow: auto` and get rid of `white-space: nowrap` since we found that `max-width` is the one causing the issue. Do you think that will work?